src/main/interface/tramp_protocol.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <string.h>
+#include "common/utils.h"
+#include "interface/tramp_protocol.h"
+
+#define TRAMP_SYNC_START                0x0F
+#define TRAMP_SYNC_STOP                 0x00
+#define TRAMP_COMMAND_SET_FREQ          0x46 // "F"
+#define TRAMP_COMMAND_SET_POWER         0x50 // "P"
+#define TRAMP_COMMAND_ACTIVE_STATE      0x49 // "I"
+#define TRAMP_COMMAND_GET_CONFIG        0x76 // "v"
+#define TRAMP_FRAME_LENGTH              16
+#define TRAMP_HEADER_LENGTH             2
```

Comment:
This should be defined as a struct and then used as `sizeof(trampHeader_t)`

Reply:
Done